package.json

```diff
@@ -1,6 +1,7 @@
 {
   "private": true,
-  "devDependencies": {
-    "@percy/agent": "~0"
+  "devDependencies": {},
+  "dependencies": {
+    "@percy/cli": "^1.0.0-beta.48"
```

Comment:
Doesn't technically matter since this is a private package, but this should technically be a dev dep.

technically